Adding Help/ Question template for issues (#366)
fastify/help: .github/ISSUE_TEMPLATE/help.md

---
name: Help

> about: If you have any questions regarding fastify this is the right place to ask

Reviewer: ```suggestion
about: If you have any questions regarding Fastify this is the right place to ask
```

title: Help
labels: help wanted
assignees: ''

---

<!--

> Before you submit an a question we recommend you first look into our  repo's issue section to see if your question have already been answered, we don't recommend duplicates.

Reviewer: ```suggestion
Before you submit a question we recommend you first look at our issue section to see if your question have already been answered.
```

"issue section" should be a link.

---

<!--

> Before you submit a question we recommend you first look at our issue section here https://github.com/fastify/help/issues  to see if your question have already been answered.

Reviewer: ```suggestion
Before you submit a question we recommend you first look at our issue section (https://github.com/fastify/help/issues) to see if your question has already been answered.
```
